velox/tpch/gen/dbgen/build.cpp

```diff
@@ -38,7 +38,7 @@ namespace facebook::velox::tpch::dbgen {
 #define V_STR(avg, seed, tgt) \
   tpch_a_rnd((int)(avg * V_STR_LOW), (int)(avg * V_STR_HGH), seed, tgt)
 #define TEXT(avg, seed, tgt) \
-  dbg_text(tgt, (int)(avg * V_STR_LOW), (int)(avg * V_STR_HGH), seed)
+  dbg_text(tgt, static_cast<int>((avg) * V_STR_LOW), static_cast<int>((avg) * V_STR_HGH), seed)
```

Comment:
Please check if the casts are already addressed by the PRs that fix the C-syle casts (see open PRs for those).

Please also fix the prior macro that has the same issue for the `avg` argument.

Reply:
Fixed the prior macro. I checked and some c style casts were resolved but these casts were not addressed.

Reply:
I believe this can be removed since it looks like it has been done in this PR
https://github.com/facebookincubator/velox/pull/11688/files